app/_src/gateway-operator/get-started/konnect/create-route.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Create a Route
+content-type: tutorial
+book: kgo-konnect-get-started
+chapter: 3
+---
+
+{:.note}
+> The following examples require you to have `jq` installed on your machine. You can use the [Konnect UI](https://cloud.konghq.com) to configure services and routes if you cannot install `jq`.
+
+Your `DataPlane` should now be running, but there are no routing rules to tell {{ site.base_gateway }} how to proxy traffic. In this tutorial we will use the Konnect API to configure and test routing.
+
+## Prerequisites
+
+1. Visit the Gateway Manager section of Konnect and fetch your control plane ID. This is available on the _Data Plane Nodes_ page next to the _Control Plane ID_ label. Export a variable that contains this value:
```

Comment:
1. In Gateway Manager, navigate to the **Overview** page of the control plane to which you added the data plane.
1. Copy the ID of the control plane in the **About this Hybrid Control Plane** section.
1. Set the value of the variable `CP_UUID` to the ID of the control plane that you copied.